quickstarts/microsoft.mobilenetwork/mobilenetwork-create-full-5gc-deployment/main.bicep

```diff
@@ -193,8 +193,8 @@ resource exampleSimPolicy 'Microsoft.MobileNetwork/mobileNetworks/simPolicies@20
   }
 }
 
-resource exampleSimGroupResource 'Microsoft.MobileNetwork/simGroups@2022-04-01-preview' = {
-  name: simGroupName
+resource exampleSimGroupResource 'Microsoft.MobileNetwork/simGroups@2022-04-01-preview' = if (!empty(simGroupName)) {
+  name: empty(simGroupName) ? 'placeHolderForValidation' : simGroupName
   location: location
```

Comment:
Resource deployment based on condition is not working as expected. Here we have addressed the issue with a workaround - [Conditional Resource Creation should skip validation](https://stackoverflow.microsoft.com/questions/281094)